sample/src/components/emulator.tsx

```diff
@@ -1,23 +1,25 @@
-import React, { FunctionComponent } from 'react'
-import ReactGbaJs from 'react-gbajs'
+import React, { FunctionComponent } from 'react';
```

Comment:
Can you remove all these lint changes on the files, please?
It's harder to know the relevant changes for your PR, and it isn't following the standard of this project.

(I forgot to add here, but the `react-gbajs` lint rules are the same as [`klo-gba.js`](https://github.com/macabeus/klo-gba.js/blob/master/brush/.eslintrc.js))

Reply:
If you want, you can add the `klo-gba.js` lint rules on this project, and fix legacy files (like `mmu.js`).

But please, on a separated PR. Otherwise, it'll almost impossible to review properly this PR